$NetBSD: patch-configure,v 1.2 2015/03/07 22:40:48 rodent Exp $

Use -Wl,-R instead of just -R to avoid breaking the build on FreeBSD.
From Dennis Lindroos in PR 49375.

--- configure.orig	2014-02-11 15:38:00.000000000 +0000
+++ configure
@@ -8047,7 +8047,7 @@ fi
   fi
   if test "X$ltrpathdirs" != "X"; then
             for found_dir in $ltrpathdirs; do
-      LTLIBICONV="${LTLIBICONV}${LTLIBICONV:+ }-R$found_dir"
+      LTLIBICONV="${LTLIBICONV}${LTLIBICONV:+ }-Wl,-R$found_dir"
     done
   fi
 
@@ -8837,7 +8837,7 @@ fi
   fi
   if test "X$ltrpathdirs" != "X"; then
             for found_dir in $ltrpathdirs; do
-      LTLIBINTL="${LTLIBINTL}${LTLIBINTL:+ }-R$found_dir"
+      LTLIBINTL="${LTLIBINTL}${LTLIBINTL:+ }-Wl,-R$found_dir"
     done
   fi
 
